temporian/core/test/compilation_test.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Tuple
+from absl.testing import absltest
+
+import temporian as tp
```

Comment:
If possible, I would like to avoid calling this main API function in any of the files (including unit tests). 
If a unit test call "import temporian as tp", essentially this is not a unit test anymore (as it loads the entire temporian lib; possibly something very large).